README.md

```diff
@@ -65,9 +68,28 @@ The view has a "Refresh" button that can be used to manually trigger a refresh o
 
 ![Camel-K integrations view Refresh](images/camelk-integrations-view-refresh-action.jpg)
 
+## Camel-K Extension Settings
+
+To access the new extension settings, go to File->Preferences->Settings, then select "Extensions" and finally "Camel-K Integration Settings."
+
+![Camel-K integrations view Settings](images/camelk-integrations-view-settings.jpg)
+
+* Proxy Namespace - Currently this drop-down has two values: default and syndesis. If we need to make this editable down the line, we can, but this was a good place to start.
+* Proxy URL - This setting corresponds to the server proxy for your Kubernetes service. It defaults to http://localhost:8000, but can be altered to any appropriate service URL. (See [Use an HTTP Proxy to Access the Kubernetes API)[https://kubernetes.io/docs/tasks/access-kubernetes-api/http-proxy-access-api/] for details on how to create the local proxy (i.e. 'kubectl proxy –port=8000') and "Starting a local Kubernetes proxy" below.)
+* Use Proxy - This setting determines whether the Camel-K Integrations view retrieves the list of running integrations via the local 'kubectl' application or via the Kubernetes Rest API and calls through the Proxy URL/Namespace combination above. The ultimate URL becomes [proxyurl]/apis/camel.apache.org/v1alpha1/namespaces/[namespace]/integrations. For now this is only used when retrieving the integrations list, but it will also be used when starting and stopping integrations when the "Use Proxy" setting is set to true.
+
+## Starting a local Kubernetes proxy
+
+We have created a new command available in the command palette (Ctrl+Shift+P or F1) called "Camel-K: Start the kubectl proxy server." This will start a new Kubernetes proxy using the Minikube executable ('kubectl proxy --port=8000) and refreshes the Camel-K Integrations view immediately. This command creates a local proxy at http://localhost:8000, which is the default value for Proxy URL in the extension settings.
```

Comment:
No, there is no auto-install of Minikube that goes with this. It would be too costly resource-wise and I think not a good decision. This is more for a local development case and I'll add that as a note in the readme.

Reply:
I'm not sure that I was clear enough.

I did not meant to have a Kubernetes instance started locally using Minikube. I meant, that we need to have the kubectl binary available on the command-line to have the proxy creation which is called through `kubectl proxy --port=8000`
I looked to the Kubernetes VS Code extension and they are [requiring to have kubectl installed locally](https://code.visualstudio.com/docs/azure/kubernetes#_before-you-begin), so even if it is not the best user experience, i think we can expect that users playing with Camel-K will also have it.